components/motor/gpio/basic.go

```diff
@@ -295,15 +298,15 @@ func (m *Motor) IsMoving(ctx context.Context) (bool, error) {
 
 // GoTo is not supported.
 func (m *Motor) GoTo(ctx context.Context, rpm, positionRevolutions float64, extra map[string]interface{}) error {
-	return errors.New("not supported")
+	return motor.NewGoToUnsupportedError(m.motorName)
```

Comment:
:+1: Thanks for creating/using these helper functions to make more standardized errors!